package.json

```diff
@@ -61,6 +61,11 @@
     "@babel/core": "^7.7.2",
     "@babel/preset-env": "^7.7.1",
     "@babel/preset-react": "^7.7.0",
+    "@rollup/plugin-babel": "^6.0.4",
```

Comment:
the older versions of rollup plugins were in the format `rollup-plugin-xxxxx`, it has now changed to `@rollup/plugin-xxxxx